setup.py

```diff
@@ -53,6 +53,6 @@
     setup_requires=["cffi"],
     ext_package="metaswitch.common",
     cffi_modules=["cffi_build.py:ffi"],
-    install_requires=["py-bcrypt", "pycrypto", "pyzmq", "cffi"],
+    install_requires=["py-bcrypt", "pycrypto", "pyzmq", "cffi", "monotonic"],
```

Comment:
I don't think we need to pin version as this module just implements the Python spec'd interface - I'd like to get bug fixes.

Good spot on the doc.